src/ocean/core/SmartUnion.d

```diff
@@ -306,7 +307,7 @@ version ( UnitTest )
 
 *******************************************************************************/
 
-private const handleInvalidCases = "case none: assert(false);" ~
+private const handleInvalidCases = "case none: verify(false);" ~
```

Comment:
Then you need a break, because DMD cannot detect that `verify` never returns

Reply:
Ah right.